### ISSUE TITLE
bug with minmaxslider: min max input broken when free: false

### ISSUE THREAD

**Issue by laurentopia** [TylerTemp/SaintsField#94]
shareX -> mp4 broke so i can't share video but here are the steps:

![image](https://github.com/user-attachments/assets/7be7ca79-3308-4103-b2d7-37a82af9e775)
![image](https://github.com/user-attachments/assets/6f7bcf3f-26d1-4d8e-9aec-d3c8fd792a50)

**Comment by TylerTemp** [comment]
Can you provide:

1.  Your unity version
2.  a simple code that may re-produce it

As my testing:

1.  Unity 2022.3.0f1 with UI Toolkit disabled.
2.  SaintsField 3.4.9
3.  Code as

```csharp
[MinMaxSlider(5, 10, free: true), BelowRichLabel("$" + nameof(rangeInt))] public Vector2Int rangeInt;
[MinMaxSlider(0, 10), BelowRichLabel("$" + nameof(rangeInt2))] public Vector2Int rangeInt2;
```

gives

https://github.com/user-attachments/assets/4d69632e-aba1-4abc-8896-2bf68417d813

**Comment by laurentopia** [comment]
![image](https://github.com/user-attachments/assets/8dc075ea-fe20-4aaf-9797-c7003bc9d4ea)
in unity 2020.3.48

**Comment by TylerTemp** [comment]
Hi,

please check if it's been fixed in [3.4.10](https://github.com/TylerTemp/SaintsField/releases/tag/3.4.10)

**Comment by laurentopia** [comment]
yay victory!
